var/spack/repos/jcsda-emc-bundles/packages/base-env/package.py

```diff
@@ -18,6 +18,9 @@ class BaseEnv(BundlePackage):
 
     version("1.0.0")
 
+    variant("python", default=True, description="Include Python")
```

Comment:
The Python variant doesn't make sense to me. There is a `depends_on(Python)` further down, which currently is true wether this variant is set or not. I'd remove line 21.